src/concat.js

```diff
@@ -30,8 +30,11 @@ var toString = require('./toString');
  *      R.concat('ABC', 'DEF'); // 'ABCDEF'
  */
 module.exports = _curry2(function concat(a, b) {
+  if (a == null || !is(Function, a.concat)) {
```

Comment:
Ah yes, I knew there was a better way to check for a function. Also, there was a slight issue where `isFunction` was named `isNumber` so I fixed that too.